fido2/client.py

```diff
@@ -262,29 +263,51 @@ def sign(self, app_id, challenge, registered_keys, timeout=None, on_keepalive=No
         }
 
 
-@unique
-class WEBAUTHN_TYPE(six.text_type, Enum):
-    MAKE_CREDENTIAL = "webauthn.create"
-    GET_ASSERTION = "webauthn.get"
+_CTAP1_INFO = b"\xa2\x01\x81\x66\x55\x32\x46\x5f\x56\x32\x03\x50" + b"\0" * 16
 
 
-_CTAP1_INFO = b"\xa2\x01\x81\x66\x55\x32\x46\x5f\x56\x32\x03\x50" + b"\0" * 16
+def running_windows_non_admin():
+    """
+    Determine if running on Windows with version >= 1903 
+    and not as an administrator, which would require the use of
+    the built in WebAuthN API in Windows.
+    """
+    import platform
+    if platform.system().lower() == 'windows':
+        import sys
+        import ctypes
+        shell = ctypes.WinDLL('shell32')
+        build = sys.getwindowsversion().build
+
+        # Corresponds to the breaking 1903 update
+        # https://en.wikipedia.org/wiki/Windows_10_version_history#Rings
+        if not shell.IsUserAnAdmin() and build >= 18362:
+            return True
 
+    return False
 
 class Fido2Client(object):
     def __init__(self, device, origin, verify=verify_rp_id):
         self.ctap1_poll_delay = 0.25
         self.origin = origin
         self._verify = verify
         try:
-            self.ctap2 = CTAP2(device)
-            self.info = self.ctap2.get_info()
+            if running_windows_non_admin:
+                self._do_make_credential = self._webauthn_make_credential
+                self._do_get_assertion = self._webauthn_get_assertion
+                self.ctap2 = WebAuthN()
+                self.info = self.ctap2.get_info()
+            else:
+                self._do_make_credential = self._ctap2_make_credential
+                self._do_get_assertion = self._ctap2_get_assertion
+                self.ctap2 = CTAP2(device)
```

Comment:
I would set `ctap2` first since it is where the exception will be thrown if there is going to be one so that the other attributes arn't being set to one value then the other.

```suggestion
                self.ctap2 = CTAP2(device)
                self._do_make_credential = self._ctap2_make_credential
                self._do_get_assertion = self._ctap2_get_assertion
```